csharp-basic-topics/ContainsMethodInCsharp/ContainsMethod/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿#region LINQ Method Syntax
+
+var isExist = false;
+var countries = new string[] { "USA", "England", "Germany", "Sweeden", "Poland" };
+
+if (countries.Contains("Sweeden"))
+    isExist = true;
+else
+    isExist = false;
+
+Console.WriteLine(isExist);
+
+#endregion
+
+#region LINQ Method Syntax
+
+var articles = new string[] { "article-1", "article-2", "article-3", "article-4", "article-5" };
+
+var isFounded = from article in articles
+                where article.Contains("article-1")
+                select article;
+
+foreach (var found in isFounded)
+{
+    Console.WriteLine("Existed article: " + found);
+}
+
+#endregion
+
+#region Manually
+
+var result = find("Code-Maze", "If you want to read great articles, then let's check the Code-Maze.");
+Console.WriteLine("Result: " + result);
+
+static (int, bool) find(string givenString, string templateText)
+{
```

Comment:
Needs to be in another class, and not static in order to be tested properly. Actually all you did in the article should be in a separate class to be properly tested in the tests. Please refactor this and make sure to use this approach in the future, because the tests you used here don't guarantee the application is running as expected.